assets/css/_route_picker.scss

```diff
@@ -0,0 +1,52 @@
+.m-route-picker {
+  background-color: $light-periwinkle;
+  height: 100%;
+  overflow-y: scroll;
+  padding: 1rem;
+  position: fixed;
+  width: 7.5rem;
+  z-index: 2;
+}
+
+.m-route-picker__selected-routes,
+.m-route-picker__route-list {
+  list-style-type: none;
+  padding: 0;
+}
+
+.m-route-picker__selected-routes {
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 2rem;
+  margin-top: 0;
+  min-height: 2rem;
+}
+
+.m-route-picker__selected-routes-button,
+.m-route-picker__route-list-button {
+  border: none;
+  border-radius: 4px;
+  color: $dark;
+  font-family: $font-family;
+  font-size: 0.8125rem;
+  font-weight: bold;
+  padding: 0.3125rem;
+}
+
+.m-route-picker__selected-routes-button {
+  background-color: $sunflower;
+  margin-bottom: 0.25rem;
+  margin-right: 0.25rem;
+  min-width: 2rem;
+}
+
+.m-route-picker__route-list-button {
+  background-color: transparentize($color: $white, $amount: 0.5);
+  margin-bottom: 0.25rem;
+  width: 100%;
+  word-break: break-word;
+
+  &--selected {
```

Comment:
I have a weak personal preference for explicitly spelling out `.m-route-picker__route-list-button--selected` because it makes grepping the codebase easier.

Same goes for the javascript where this class is assembled.

Reply:
I can see that. On the other hand this style is arguable more readble and easier to understand the intention.